.github/PULL_REQUEST_TEMPLATE/pr_chore.md

```
@@ -0,0 +1,41 @@
+name: "Chore PR"
+about: "Repo hygiene: configs, scripts, formatting — no behaviour change"
+title: "chore: <scope>"
+labels: ["status:needs-review"]
+---
+<!-- Note: YAML front matter is parsed by our labeler workflow; GitHub itself does not apply labels from front matter. -->
+
+<!-- Suggested additional labels (pick as relevant):
+- area:dependencies or area:ci (if relevant)
+-->
+
+## Summary
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)